Model/abstractShip.php

```diff
@@ -93,14 +99,14 @@ public function setWeaponPower($weaponPow)
         $this->weaponPower = $weaponPow;
     }
 
-    public function setStrength($strength)
+    public function setMaxHealth($maxHealth)
     {
-        $this-> strength = $strength;
+        $this-> maxHealth = $maxHealth;
```

Comment:
Remove the space between $this-> and maxHealth

Reply:
Done